projects/bst/README.md

```diff
@@ -0,0 +1,23 @@
+## Blended Skill Talk
+
+### _Can You Put it All Together: Evaluating Conversational Agents' Ability to Blend Skills_
+
+Eric Michael Smith, Mary Williamson, Kurt Shuster, Jason Weston, Y-Lan Boureau
```

Comment:
Ha, we can put the asterisks by my name and Mary's name :)

Reply:
done